examples/lib/defaultHtmlStepUi.js

```diff
@@ -15,8 +15,6 @@ const intermediateHtmlStepUi = require('./intermediateHtmlStepUi.js'),
   scopeQuery = require('./scopeQuery');
 
 function DefaultHtmlStepUi(_sequencer, options) {
-  let $step, $stepAll;
```

Comment:
Just checking, why you deleted this line?
I think you did it accidently

Reply:
Oops. That is actually broken code which will be fixed in Keshav's PR. IDK how I managed to delete it.

Reply:
Removing those lines wouldn't cause errors though. The code will work exactly the same. The only issue is that the code itself produces unexpected output.